packaging/systemd/configure-ovs.sh

```diff
@@ -626,6 +626,23 @@ fi
 # print initial state
 print_state
 if [ "$1" == "OVNKubernetes" ]; then
+  # Skip configuring NICs onto OVS bridge "br-ex" when disableOVSInit is true
+  MICROSHIFT_CONFIG_DIRS=(~/.microshift /etc/microshift)
```

Comment:
Is it likely that a systemd unit would find something in ~/.microshift? Should we just look in /etc?

Reply:
Good catch!
agree, systemd may not find the config file in user root. I'll update to use /etc/ directly.

Reply:
done.